tests/fips/fips_setup.pm

```diff
@@ -21,17 +21,22 @@ use testapi;
 use strict;
 
 # Install fips pattern and update grub.cfg to boot with fips=1
-sub run() {
+sub run {
     select_console 'root-console';
 
-    if (!script_output "grep 'fips=1' /proc/cmdline | tee") {
-        script_run 'zypper -n install --type pattern fips', 300;
```

Comment:
Seems it won't work. If fips is not enabled on target, then:
script_output "! grep 'fips=1' /proc/cmdline" will return empty string.
Then afterward commands won't be run, which is actually required to install fips pattern.